content/200-concepts/100-components/02-prisma-client/059-embedded-types.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: 'Embedded documents'
+metaTitle: 'Embedded documents (Preview)'
+metaDescription: 'This page explains how to use embedded documents.'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+<Admonition type="info">
+
+Embedded documents are available in Preview in versions 3.10.0 and later.
+
+</Admonition>
+
+<Admonition type="warning">
+
+Embedded documents are currently only available on MongoDB.
```

Comment:
Should probably have something about how to enable this. Preview will be short for this feature though (~6wks), so this will be temporary

Similar to: https://deploy-preview-2858--prisma2-docs.netlify.app/docs/concepts/components/prisma-client/full-text-search#enabling-full-text-search

Reply:
Thanks, have added this